docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,211 @@
+---
+id: dual-region
+title: "Dual region"
+sidebar_label: "Dual region"
+description: "A dual-region setup allows you to run Camunda in two regions synchronously."
+---
+
+import DualRegion from "./img/dual-region.svg";
+
+Camunda 8 is compatible with a dual-region setup under certain [limitations](#limitations). This allows Camunda 8 to run in a mix of active-active and active-passive setups, resulting in an overall **active-passive** setup. The following will explore the concept, limitations, and considerations.
+
+:::warning
+
+You should get familiar with the topic, the [limitations](#limitations) of the dual-region setup, and the general [considerations](#considerations) on operating a dual-region setup.
+
+:::
+
+## Active-active and active-passive
+
+**Active-active** and **active-passive** are standard setups used in dual region configurations to ensure that applications remain available and operational in case of failures.
+
+In an **active-active** setup, multiple application instances run simultaneously in different regions, actively handling user requests. This allows for better load balancing and fault tolerance, as traffic can spread across regions. If one region fails, the workload can shift to another without causing disruptions.
+
+By contrast, an **active-passive** setup designates one region as the main or active region where all user requests are processed. The other region remains on standby until needed, only becoming active if the previously active region fails. This setup is easier to manage but may result in higher delays during failover events.
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test [operational procedures](<!-- TODO: Link -->) in non-production environments based on the framework steps outlined by Camunda **before applying them in production setups**.
+- Before advancing to production go-live, customers need to validate these procedures with Camunda.
```

Comment:
The [current proposal](https://camunda.slack.com/archives/C0486FPV076/p1708592171114499) is not to make it compulsory. So we can re-phrase it as "It is strongly recommended" instead of "need to".